src/overlay.js

```diff
@@ -19,34 +19,38 @@ function applyOverlayToOpenAPI(spec, overlay) {
 			}
 
 		} else {
-			// It must be an update
-			jsonpath.apply(spec, a.target, (chunk) => {
-				
-				// Deep merge using a module (built-in spread operator is only shallow)
-				const merger = mergician({appendArrays: true});
-				const merged = merger(chunk, a.update);
-				return merged;
+			try {
+				// It must be an update
+				jsonpath.apply(spec, a.target, (chunk) => {
+					// Deep merge using a module (built-in spread operator is only shallow)
+					const merger = mergician({appendArrays: true})
+					return merger(chunk, a.update)
+				});
+			}
+			catch (ex) {
+				process.stderr.write(`Error applying overlay: ${ex.message}\n`)
+				//return chunk
+			}
 
-			});
 		}
 	})
 
 	return spec;
 }
 
 function sortOpenAPIFields(field1, field2) {
-    const orderedKeys = ["info", "servers", "paths", "components", "summary", "objectId", "description", "tags", "parameters", "responses"];
+	const orderedKeys = ["info", "servers", "summary", "operationId", "tags", "paths", "components", "description", "parameters", "responses"];
```

Comment:
Thanks, I was hoping someone had thought about this more than I had!

Reply:
It's a bit awkward because tags can appear more than once but there's only one sort precedence for it...